### PR TITLE
download tsv button

### PR DESCRIPTION
**Pull request by abgeorge7**
Description about what this pull request does.

Added a "download all" button to our query page so a user can download all tsvs for a given node in a program/project.

### New Features
- Implemented XXX

### CODE REVIEW

**Comment by PlanXCyborg** [comment]
Jenkins Build 1 : time taken 3 min 25 sec

File: src/QueryNode/QueryForm.jsx [hunk 1]
```diff
@@ -9,14 +12,26 @@ class QueryForm extends React.Component {
...
     };
     this.updateValue = this.updateValue.bind(this);
+    this.handleDownloadAll = this.handleDownloadAll.bind(this);
     this.handleQuerySubmit = this.handleQuerySubmit.bind(this);
   }
 
+  handleDownloadAll() {
+    const programProject = this.props.project.split('-');
+    window.open(
+      `${submissionApiPath}${programProject[0]}/${programProject[1]}/export?node_label=${this.state.selectValue.value}&format=tsv`,
```
Comment (paulineribeyre) [inline src/QueryNode/QueryForm.jsx]: the project code could contain dashes, could you add this? https://github.com/uc-cdis/data-portal/blob/master/src/utils.js#L6-L8

Comment (abgeorge7) [reply src/QueryNode/QueryForm.jsx]: :O thanks!

**Comment by PlanXCyborg** [comment]
Jenkins Build 2 : time taken 14 min

File: src/QueryNode/QueryForm.jsx [hunk 2]
```diff
@@ -50,6 +64,29 @@ class QueryForm extends React.Component {
         <Select className='query-form__select' name='node_type' options={options} value={state.selectValue} onChange={this.updateValue} />
         <input className='query-form__input' placeholder='submitter_id' type='text' name='submitter_id' />
         <input className='query-form__search-button' type='submit' onSubmit={this.handleQuerySubmit} value='search' />
+        {
+          this.state.selectValue ? (
```
Comment (mfshao) [inline src/QueryNode/QueryForm.jsx]: Maybe a bit more to ask but is it easy to add some logic here to let the dropdown button only shows when there are nodes in the query page?

Currently the `Download All` button will show even if there is no nodes under a `node_type`, which is a bit weird.

**Comment by PlanXCyborg** [comment]
Jenkins Build 3 : time taken 46 min

**Comment by PlanXCyborg** [comment]
Jenkins Build 4 : time taken 1 hr 6 min

**Comment by PlanXCyborg** [comment]
Jenkins Build 6 : time taken 38 min

**Comment by PlanXCyborg** [comment]
Jenkins Build 7 : time taken 48 min

**Pull request merged**